test/lib/cql_test_env.cc

```diff
@@ -161,6 +162,7 @@ class single_node_cql_env : public cql_test_env {
     sharded<locator::effective_replication_map_factory> _erm_factory;
     sharded<sstables::directory_semaphore> _sst_dir_semaphore;
     std::optional<utils::disk_space_monitor> _disk_space_monitor_shard0;
+    std::optional<replica::out_of_space_controller> _out_of_space_controller_shard0;
```

Comment:
Maybe we should merge disk_space_monitor and out_of_space_controller. @bhalevy ?

Reply:
sounds like a good idea